src/fiatconnect/types.ts

```diff
@@ -0,0 +1,11 @@
+import { FiatAccountSchema, FiatAccountSchemas } from '@fiatconnect/fiatconnect-types'
+
+export interface FiatConnectSchemaCountryOverrides<T extends FiatAccountSchema> {
+  [country: string]: {
+    [Property in keyof Partial<FiatAccountSchemas[T]>]: {
+      regex?: string
```

Comment:
Should `regex` and `errorString` be optional? I would assume we would always want `regex`, and I assume that we would never want to show the default error message when using a custom regex?

Reply:
I left them optional in case in the future we override things other than regex, like label. But we don't have a case so I can make them required for now. Related, should we also allow overriding keyboardType right now, in case we wanted to allow anything other than numbers? It defaults to `numpad`

Reply:
Ah yeah, that makes sense. I suppose we can make them required for now and change to optional later if want to use this to override unrelated parameters/fields. I was thinking about `numpad` as well; I think we can leave that be until we have a usecase for overriding it.

As a sidenote, all of this remote configuration/overriding makes me wonder if we can't just have some completely remotely configurable solution for _all_ account schemas, such that supporting a new schema wouldn't require _any_ changes within the wallet; just specify default rules/overrides in firebase per schema and have the wallet construct the UI accordingly. Maybe premature right now, but a potential idea for later perhaps.